### PR TITLE
Code district

### PR DESCRIPTION
**Pull request by aliahmad45**

### CODE REVIEW

File: resources/views/topics/managetopic.blade.php [hunk 1]
```diff
@@ -107,7 +107,7 @@
            
             <div class="form-group">
                 <label for="">Edit summary (Briefly describe your changes)</label>
-                <textarea class="form-control" rows="4" name="note" id="note"> </textarea>
+                <textarea class="form-control" rows="4" name="note" id="note">{{$topicupdate == 'update'? $topic->note : ""}}</textarea>
```
Comment (akuks) [inline resources/views/topics/managetopic.blade.php:110]: I think we have used notation `$topicUpdate` insteadof `$topicupdate`.

Comment (aliahmad45) [reply resources/views/topics/managetopic.blade.php:110]: @akuks have you checked their notations used in controllers? They have used same notations for topic, statements and camps. No rules are followed in Canonizer 2.0. We will follow these notations/rules in canonizer 3.0 but for canonizer 2.0 we should use the same nations as they have already used.

Comment (aliahmad45) [reply resources/views/topics/managetopic.blade.php:110]: @akuks  these variables are coming from controllers and we need to update in everywhere where it's being used.

Comment (akuks) [reply resources/views/topics/managetopic.blade.php:110]: Thats fine in that case.

**Pull request merged**